locales/translations/es.json

```
@@ -58,6 +58,8 @@
     "activities": "registros",
     "activity": "registro",
     "Activity": "Actividad",
+    "Actual success rate": "Tasa de éxito actual",
```

```suggestion
    "Actual success rate": "Tasa de éxito real",
```

Other options would be `efectiva` or `medida`, but `real` is OK. Spanish's `actual` is English's `current`.